src/ios/ListLibraryItems.m

```diff
@@ -102,6 +100,16 @@ - (void)returnUploadResult:(BOOL)aSuccess payload:(NSDictionary *)aJSON command:
     [[self commandDelegate] sendPluginResult:pluginResult callbackId:[aCommand callbackId]];
 }
 
+- (bool)isTaskCanceledDueToAppKill:(NSURL*)aLocalTempURL error:(NSError *)anError status:(long)aStatus errormessage:(NSString *)anErrorMessage target:(NSString *)aTarget {
+    if(aLocalTempURL == nil && [anError code] == NSURLErrorCancelled) {
+        return true;
+    } else {
+        NSMutableDictionary * json = [NSMutableDictionary dictionaryWithObjects:@[[NSString stringWithFormat:@"%ld",aStatus], anErrorMessage,[aLocalTempURL absoluteString], aTarget]
+                                                                        forKeys:@[@"code", @"message", @"source", @"target"]];
+        [self returnUploadResult:NO payload:json command:mCommand];
+        return false;
```

Comment:
The else part should not be in that method. 

This method should only be responsible of the cancellation detection. Also, it should only receive `aLocalTempURL` and an `error`. No need to have the `status` or `message` since we can get it from the `error` object, right?

Reply:
Fixed, I just left the two `return` in `if()` block. Now the JSON creation part is back in the delegate function !